README.md

```diff
@@ -1 +1,65 @@
-# React-Capstone-project-Financial-modeling
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Crypto currencies
+
+In this project I have built a Crypto currency app, you can see the common currencies on the home page with some details about them, once you click on the arrow, it will take you to another page with all the information for the selected currency. All the data are fitched from binance Api.
+
+
+## Built With
+
+- Javascript
+- React
+- Redux
+
+
+## Getting Started
+
+To successfully utilize this project first of all you need to clone or download the repository and follow the setup process.
+
+## Setup
+
+- copy this link
+
+https://github.com/tarek102/react-capstone-project-financial-modeling.git
+
+- Choose the directory that you want to clone the repository.
+- Open the terminal in this directory and type the following command
+
+``` git clone https://github.com/tarek102/react-capstone-project-financial-modeling.git ```
+
+- run  ``` npm install ```
+- run ``` npm start ```
+
+
+## prerequisites:
+- Visual Studio
+- Browser
+- Github
+- Node JS
+
+
+## Author
+
+👤 Tarek
+- GitHub: [@tarek102](https://github.com/tarek102)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/tarek102/react-capstone-project-financial-modeling/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ ] In keeping with the Creative Commons requirements, kindly include appropriate credit to Nelson Sakwa whose design is being used for the project.